File: projects/order-bc/core/src/main/java/com/robinfood/core/dtos/OrderDailyDTO.java
package com.robinfood.core.dtos;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
public class OrderDailyDTO {

    private final Long brandId;

    private final LocalDateTime createdAt;

    private final Long deliveryTypeId;

    private final Long id;

    private final String orderInvoiceNumber;

    private final String orderNumber;

    private final OriginDTO origin;

    private final Double total;

    private final UserDataDTO user;

}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/OrderDailyDTO.kt
package com.robinfood.core.dtos

import java.math.BigDecimal

data class OrderDailyDTO(
    val id: Long,
    val orderNumber: String,
    val invoiceNumber: String,
    val totalPrice: BigDecimal,
    val deliveryTypeId: Int,
    val originId: Long,
    val origin: String,
    val originBackgroundColor: String,
    val brandId: Long,
    val brandImage: String,
    val brandBackgroundColor: String,
    val userId: Long,
    val customerName: String,
    val creationTime: String
)
